tests/pkg_micro-ecc/Makefile

```diff
@@ -0,0 +1,11 @@
+APPLICATION = micro-ecc
+include ../Makefile.tests_common
+
+# boards without hardware random number generator
+BOARD_BLACKLIST := arduino-mega2560 chronos msb-430 msb-430h qemu-i386 telosb wsn430-v1_3b wsn430-v1_4 z1 samr21-xpro
```

Comment:
This should fix the failure Travis experiences as pointed out in https://github.com/RIOT-OS/RIOT/pull/2236#issuecomment-73368629 btw.